packages/loaders/openapi/tests/__snapshots__/deepl.test.ts.snap

```diff
@@ -278,6 +278,7 @@ For privacy reasons the translated document is automatically removed from the se
       "field": "getGlossaryEntries",
       "headers": Object {
         "Accept": "{args.Accept}",
+        "accept": "text/tab-separated-values",
```

Comment:
@ardatan this is weird, the capitalized header is left (only) in this test

Reply:
That's because we don't normalize header names while merging root  level `operationHeaders` and operation level headers.

Reply:
https://github.com/Urigo/graphql-mesh/blob/762ad8e644335fa7baf2bacdb317314eb410be3e/packages/loaders/json-schema/src/addExecutionLogicToComposer.ts#L163
Maybe we should lowercase header names here while creating the final `headers` object